pandas/core/reshape/merge.py

```diff
@@ -2067,17 +2063,26 @@ def _validate_operand(obj: FrameOrSeries) -> "DataFrame":
         )
 
 
-def _items_overlap_with_suffix(left: Index, lsuffix, right: Index, rsuffix):
+def _items_overlap_with_suffix(left: Index, right: Index, suffixes):
```

Comment:
type suffixes Tuple[str, str]

Reply:
In my latest pull request, I have validated type for `tuple`. Should I do explicit validation for type to be `tuple(str, str)`??